src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -2,21 +2,51 @@
 Copyright (c) 2018 Patrick Massot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot, Johannes Hölzl
-
-Continuous linear functions -- functions between normed vector spaces which are bounded and linear.
 -/
 import analysis.normed_space.multilinear
 
+/-!
+# Continuous linear maps
+
+This file defines a class stating that a map between normed vector spaces is (bi)linear and
+continuous.
+Instead of asking for continuity, the definition takes the equivalent condition that `∥f x∥` is
+bounded by a multiple of `∥x∥`. Hence the "bounded" in the name refers to `∥f x∥/∥x∥` rather than
+`∥f x∥` itself.
+
+## Main declarations
+
+* `is_bounded_linear_map`: Class stating that a map `f : E → F` is linear and continuous, but
+  through the simpler to provide statement that `∥ f x ∥` is bounded by a multiple of `∥ x ∥`.
```

Comment:
Modern mathlib style says (eg) `∥ f x ∥` rather than `∥f x∥`.  (If you've seen it done another way in this file, it's because it's older.)